### ISSUE TITLE
Script with @run-at document-start will be executed twice

### ISSUE THREAD

**Issue by Infocatcher** [greasemonkey/greasemonkey#2371]
Testcase: https://gist.github.com/Infocatcher/1243f295ca93c94160ec
Install: https://gist.githubusercontent.com/Infocatcher/1243f295ca93c94160ec/raw/document-start.user.js
(and then open about:blank?UserScripts/test - should appear only one `+`)

```js
// ==UserScript==
// @name        about:blank?UserScripts/test
// @namespace   dev/null
// @include     about:blank?UserScripts/test
// @run-at      document-start
// @version     1
// @grant       none
// ==/UserScript==

console.log("test: " + document.readyState);
document.body.appendChild(document.createTextNode("+"));
```

Tested with Greasemonkey 3.7, 3.8beta2 (Firefox 44.0.2).

**Comment by arantius** [comment]
Fix merged.

**Comment by arantius** [comment]
This should be fixed in build 3.8beta3; confirmation would be helpful.

https://addons.mozilla.org/en-US/firefox/addon/greasemonkey/versions/3.8beta3

**Comment by Infocatcher** [comment]
Confirmed, looks fixed for me with GM 3.8beta3 too.
Here I've tested with custom build from master branch: https://github.com/greasemonkey/greasemonkey/pull/2372#issuecomment-211471368
Thanks again. :)
